src/connectors/infinite-hits/__tests__/connectInfiniteHits-test.ts

```diff
@@ -842,220 +843,188 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/infinite-hi
+    });
 
-        expect(uiStateAfter).toBe(uiStateBefore);
+    test('return Search Parameters without highlighted tags when `escapeHTML` is `false`', () => {
+      const render = jest.fn();
+      const makeWidget = connectInfiniteHits(render);
+      const helper = algoliasearchHelper(createSearchClient(), 'indexName');
+      const widget = makeWidget({
+        escapeHTML: false,
       });
-    });
 
-    describe('getWidgetSearchParameters', () => {
-      it('should return the same SP if there are no refinements in the UI state', () => {
-        const renderFn = jest.fn();
-        const makeWidget = connectInfiniteHits(renderFn);
-        const widget = makeWidget({ showPrevious: true });
-
-        const helper = algoliasearchHelper({} as Client, '', {});
-        helper.search = jest.fn();
-
-        widget.init!(
-          createInitOptions({
-            state: helper.state,
-            helper,
-          })
-        );
-
-        // The user presses back (browser), and the URL contains no parameters
-        const uiState = {};
-        // The current state is empty (and page is set to 0 by default)
-        const searchParametersBefore = SearchParameters.make(helper.state);
-        const searchParametersAfter = widget.getWidgetSearchParameters!(
-          searchParametersBefore,
-          { uiState }
-        );
-
-        // Applying the same values should not return a new object
-        expect(searchParametersAfter).toBe(searchParametersBefore);
+      const actual = widget.getWidgetSearchParameters!(helper.state, {
+        uiState: {},
       });
 
-      it('should enforce the default value if no value is in the UI State', () => {
-        const renderFn = jest.fn();
-        const makeWidget = connectInfiniteHits(renderFn);
-        const widget = makeWidget({ showPrevious: true });
-
-        const helper = algoliasearchHelper({} as Client, '', {});
-        helper.search = jest.fn();
-
-        widget.init!(
-          createInitOptions({
-            state: helper.state,
-            helper,
-          })
-        );
-
-        const { showMore } = renderFn.mock.calls[0][0];
-
-        // The user presses back (browser), and the URL contains no parameters
-        const uiState = {};
-        // The current state is set to next page
-        showMore();
-        const searchParametersBefore = SearchParameters.make(helper.state);
-        const searchParametersAfter = widget.getWidgetSearchParameters!(
-          searchParametersBefore,
-          {
-            uiState,
-          }
-        );
-        // Applying an empty state, should force back to page 0
-        expect(searchParametersAfter).toMatchSnapshot();
-        expect(searchParametersAfter.page).toBeUndefined();
+      expect(actual.highlightPreTag).toBeUndefined();
+      expect(actual.highlightPostTag).toBeUndefined();
+    });
+
+    test('return Search Parameters with empty page when `showPrevious` without `page` in the UI state', () => {
```

Comment:
The name of the test does not match the expectation (the same problem is present on the next cases). The page is not empty but `0`.